guardian/cdk
# feat: Use latest versions of @guardian/cdk and aws-cdk when generating projects

## What does this change?

This PR ensures that the project generation tooling always pulls in the latest `@guardian/cdk` version and the appropriate `aws-cdk` versions.

## How to test

Run `./script/cli new --app "myApp" --stack "myStack"` locally. Note that this relies on https://github.com/guardian/cdk/pull/1027 (or at the very least, temporarily hardcoding the version in the root-level `package.json`) to work properly locally.

## How can we measure success?

New projects use the latest `cdk` dependencies by default.

## Have we considered potential risks?

Yes, there is a risk that by automatically updating these library versions we'll break the project generation tooling. We're planning to add some integration tests (in a future PR) to mitigate this risk.

## Checklist

- [x] I have listed any breaking changes, along with a migration path [^1] N/A
- [x] I have updated the documentation as required for the described changes [^2] N/A

[^1]: Consider whether this is something that will mean changes to projects that have already been migrated, or to the CDK CLI tool. If changes are required, consider adding a checklist here and/or linking to related PRs.
[^2]: If you are adding a new construct or pattern, has new documentation been added? If you are amending defaults or changing behaviour, are the existing docs still valid?
